source/memory.tex

```diff
@@ -3284,7 +3284,7 @@
 
 \pnum
 In the constructor definitions below,
-enables \tcode{shared_from_this} with \tcode{p},
+\defnx{Enables \tcode{shared_from_this}}{enables \tcode{shared_from_this} with \tcode{p}},
```

Comment:
...although maybe it would be better to say
```
\defnx{Enabling \tcode{shared_from_this} with \tcode{p}}{enables \tcode{shared_from_this} with \tcode{p}},
```